Doc/library/configparser.rst

```diff
@@ -1030,6 +1030,13 @@ ConfigParser Objects
       directory), and all existing configuration files in the iterable will be
       read.
 
+      Raise :exc:`ParsingError` instead of :exc:`AttributeError`
+      when the configuration contains a key without a corresponding value.
+      This change is intended to handle scenarios where a key lacks a value
+      in a configuration which contains extra space or newline within a section.
+      Please note that it's not recommended to have extra spaces
+      or blank lines within sections of the configuration.
```

Comment:
Not sure if a general-use library should express such an opinion!

Reply:
Is a versionchaged note needed?

Reply:
okay, I will remove `Please note that it's not recommended to have extra spaces or blank lines within sections of the configuration.`

Reply:
>Is a versionchaged note needed?

No, I was advised to remove it.

Reply:
My suggestion is to describe the behavior online with the rest of the documentation. Then a short versionchanged note about the new condition that raises an exception. But do not repeat the documentation in the versionchanged note.

Reply:
okay